LICENCE

```diff
@@ -0,0 +1,19 @@
+Copyright 2020 Department for Education Digital
```

Comment:
After checking a bunch of repositories on https://github.com/DFE-Digital I found this MIT copyright is used most. 
```suggestion
MIT License

Copyright (c) 2020 Crown Copyright (Department for Education)
```

Reply:
Ok, fixed this. I copied the text directly from the [OSI](https://opensource.org/licenses/MIT).